### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by FM1337** [FM1337/FactorioCord#3]
So I have not been able to get FactorioCord to work on windows correctly. it won't write to the factorio.log and it will not allow messages to be sent or received between discord and in-game. As such I'm going ahead and releasing version 1.1 and will attempt to have Windows support in a future version. Sorry about this.

If anyone wants to take a stab at trying to fix it, help would be most appreciated thanks!

**Comment by FM1337** [comment]
Error sending discord chat to in-game

```
2017-07-30 12:29:44.0106185 -0300 ADT: An error occurred when attempting to send Discord chat to in-game
Details: write |1: The pipe is being closed.
```

**Comment by FM1337** [comment]
also

```
2017-07-30 12:43:13.7054626 -0300 ADT: An error occurred when attempting to pass input to the console
Details: write |1: The pipe is being closed.
```

**Comment by notAreYouScared** [comment]
Was just trying this on my pc and windows server both give the exact same errors. I even tried the first release, it dosnt throw an error. It just does nothing. It connects to discord but refused to relay any messages. The stop connect worked great, but only killed the bot, not the server.

**Comment by FM1337** [comment]
@cjmwid yeah, I worked on it for a few weeks trying to figure it out, and couldn't get it to work correctly on windows, and I don't have any time to continue trying to find a fix for it for now. Anyone is welcome to try to find a fix and it would be really appreciated.

**Comment by notAreYouScared** [comment]
According to one of the people I talked to in the Golang discord chat they said line 159 in main.go might be causing the issue. You know more then me thought I'd share

**Comment by zackman0010** [comment]
From what I was told by someone else trying the same thing, the headless server on Windows is not actually a console, as that version of Factorio was not built to be a console program. It is instead a GUI that mimics a console. As such, the usual STDIN/STDOUT redirects do not work. I've written my own server management program in C that uses the same process (start a new process and redirect the pipes) to allow communication with Discord, and it runs into the same problem.

**Comment by szentigrad3** [comment]
is this still been worked on?
